services/QuillLMS/app/controllers/api/api_controller.rb

```diff
@@ -35,23 +36,6 @@ class AccessForbidden < StandardError; end
     response.headers['X-API-Reference'] = 'http://docs.empirical.org/api-reference/'
   end
 
-  private def current_user
-    begin
-      if session[:user_id]
-        @current_user ||= User.find(session[:user_id])
-      elsif doorkeeper_token
-        User.find_by_id(doorkeeper_token.resource_owner_id)
-      else
-        authenticate_with_http_basic do |username, password|
-          return @current_user ||= User.find_by_token!(username) if username.present?
-        end
-      end
-    rescue ActiveRecord::RecordNotFound
-      sign_out
-      nil
-    end
-  end
-
   private def doorkeeper_token
```

Comment:
Is this method still needed?

Reply:
I think we no longer use doorkeeper, but I couldn't guarantee it off-hand.

Reply:
QuillAuthentication already has a doorkeeper_token method, so I think the method here is redundant.

Reply:
@brendanshean I'll remove it, thanks for flagging.